### DISCUSSION TITLE
Freeze local offset

### DISCUSSION THREAD

**Discussion by developomp** [ppy/osu#17766] [Ideas]
Remembering which beatmaps have a perfect local offset can be quite difficult, and recalibrating one after accidentally overriding it takes a bit of time. Having a feature that freezes local offset for  a specific beatmap would solve this problem.

**Comment by peppy** [comment]
I don't get it. Just don't click the button?

**Reply by developomp** [reply]
I wish I didn't

**Reply by developomp** [reply]
Ohhhh now I see what you mean.
A play can be off by a couple of milliseconds even when the local offset is perfect, and deciding whether the inaccuracy is due to a misconfiguration or just bad luck can be difficult without a way to “label” maps with a good local offset.

**Comment by frenzibyte** [comment]
Safe to say this goes hand-in-hand with https://github.com/ppy/osu/discussions/17652#discussioncomment-2506688. Unsure about the direction however.
